common/lib/xmodule/xmodule/js/src/video/01_initialize.js

```diff
@@ -131,16 +131,12 @@ function (VideoPlayer, i18n, moment) {
             state.youtubeApiAvailable = false;
 
             onYTApiReady = function () {
-                console.log('[Video info]: YouTube API is available and is loaded.');
```

Comment:
I have the same concern here regarding the removal of log info.  These seem like they were intentionally put here and removing them is unrelated to the other work in this PR.  Is it possible we are somehow tracking this logged information for certain use cases?